views/_includes/feedback.njk

```diff
@@ -0,0 +1,50 @@
+<details class="w-full sm:w-2/3 lg:w-1/2">
+    <summary class="px-4 py-2 bg-gray-300 rounded inline-block">{{ __('feedback.button') }}</summary>
+    
+    <form action="/{{ getLocale() }}/feedback" method="post" class="px-8 py-6 rounded border border-gray-300 bg-gray-100" id="feedback-form">
```

Comment:
This action path is like this because we need to submit to a language-specific url so we can capture the user's selected language, but the Feedback form renders on the 404 and 500 pages, and the routePath helper is not available on those pages.